site/content/zh/contributions/DOCS.md

```diff
@@ -7,6 +7,8 @@ description: "本节讲述 Envoy Gateway 文档的开发工作。"
 我们已将 Envoy Gateway 文档从 **Sphinx** 迁移到 **Hugo**。
 {{% /alert %}}
 
+阅读博客：[欢迎来到新网站！（英文）](/news/blogs/new-website/new-website)
```

Comment:
It's been a while since the EG website is migrated to hugo. I suggested we remove the Notice on the top of this page. WDYT?

Reply:
Agree, let me do it for both content.

Reply:
Done